docs/api-docs.asciidoc

```diff
@@ -3,24 +3,28 @@
 == API Reference
 
 .Packages
+* xref:{p}-apm-k8s-elastic-co-v1[apm.k8s.elastic.co/v1]
+* xref:{p}-common-k8s-elastic-co-v1[common.k8s.elastic.co/v1]
 * xref:{p}-apm-k8s-elastic-co-v1beta1[apm.k8s.elastic.co/v1beta1]
 * xref:{p}-common-k8s-elastic-co-v1beta1[common.k8s.elastic.co/v1beta1]
+* xref:{p}-elasticsearch-k8s-elastic-co-v1[elasticsearch.k8s.elastic.co/v1]
+* xref:{p}-kibana-k8s-elastic-co-v1[kibana.k8s.elastic.co/v1]
 * xref:{p}-elasticsearch-k8s-elastic-co-v1beta1[elasticsearch.k8s.elastic.co/v1beta1]
 * xref:{p}-kibana-k8s-elastic-co-v1beta1[kibana.k8s.elastic.co/v1beta1]
```

Comment:
The order of the packages is a little strange but I'm not sure that we can do anything for that easily and quickly.

Reply:
Fixed with https://github.com/elastic/gen-crd-api-reference-docs/pull/1